agent/util-scripts/unittests

```diff
@@ -531,6 +534,7 @@ declare -A pre_hooks=(
     [test-06]='ln -s mock-pbench-tool-meister-client ${_testopt}/unittest-scripts/pbench-tool-meister-client; mkdir -p ${_testdir}/42-iter/sample42/tools-default/testhost.example.com/iostat; touch ${_testdir}/42-iter/sample42/tools-default/testhost.example.com/iostat/iostat-stdout.txt'
     [test-07]='mkdir -p ${_testdir}/42-iter/sample42/tools-foobar/testhost.example.com/iostat; touch ${_testdir}/42-iter/sample42/tools-foobar/testhost.example.com/iostat/iostat-stdout.txt'
     [test-11.16]='mkdir -p ${_testdir}/tmp; (echo foo; echo bar) > ${_testdir}/tmp/good-remote-file'
+    [test-11.17]='mkdir -p ${_testdir}/tmp; touch  ${_testdir}/tmp/empty-remote-file'
```

Comment:
Shouldn't there be a "post hook" which removes this file again?  (Aside from being tidy, this prevents it from becoming non-empty in the future....)

Reply:
`_reset_state` deletes the whole `_testdir`  after each test (which is why the file was created in `${_testdir}/tmp`). The post hooks are meant for things like sorting output to make it reproducible.

Reply:
Cool.  (A recent PR from @portante, included a post-hook clean-up, so I was keying off that.)